src/features/pages/partners/deriv-prime/commercial-plan/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { commercial_table } from './styles.module.scss'
+import Container from 'features/components/atoms/container'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import Table from 'features/components/atoms/table'
+import dclsx from 'features/utils/dclsx'
+
+const tableData = [
+    {
+        '_t_Monthly volume requirement_t_': <Localize translate_text="_t_0 - 3 billion_t_" />,
+        '_t_Fee per million_t_': <Localize translate_text="_t_$ 8.00_t_" />,
+    },
+    {
+        '_t_Monthly volume requirement_t_': <Localize translate_text="_t_> 3 - 5 billion_t_" />,
```

Comment:
Can you format the data structure like `column_data` with all the column values, and `row_data`with all the row values than repeating the Monthly volume requirement and fee per million many times

Reply:
Done...